vendor/assets/javascripts/modernizr-custom.js

```diff
@@ -0,0 +1,255 @@
+/*!
```

Comment:
This is an old version of Modernizr. We should use the latest release: https://github.com/Modernizr/Modernizr/releases

And for actual use by apps, we do want the minified version. Using an unminified version should only be done if debugging something and not committed for use in production.

And could you document the steps used to "build" it so others can update easily in the future? Probably in `/docs/installation.md`

Reply:
I have updated it to the latest version and updated the docs.